packages/cli/package.json

```diff
@@ -38,7 +38,10 @@
     "lint": "prettier --write ./src",
     "format": "prettier --write ./src",
     "format:check": "prettier --check ./src",
-    "clean": "rm -rf dist .turbo node_modules .turbo-tsconfig.json tsconfig.tsbuildinfo"
+    "clean": "rm -rf dist .turbo node_modules .turbo-tsconfig.json tsconfig.tsbuildinfo",
+    "test:setup-commands": "vitest test/setup-commands.test.js",
```

Comment:
we should probably create a command to run each of these with the && sign on so like a test:cli one